Time.php

```diff
@@ -15,7 +15,7 @@
 	return 1;
 }
 
-define( 'DATAVALUES_TIME_VERSION', '0.8.0 alpha' );
+define( 'DATAVALUES_TIME_VERSION', '0.7.1 alpha' );
```

Comment:
That's on purpose. This is an added feature, not a breaking change, and can go in a minor release. We can increase this later if it turns out all changes should go into 0.8.